pkg/server/server.go

```diff
@@ -72,23 +68,8 @@ func NewSSIServer(shutdown chan os.Signal, cfg config.SSIServiceConfig) (*SSISer
 	// service-level routers
 	engine.GET(HealthPrefix, router.Health)
 	engine.GET(ReadinessPrefix, router.Readiness(ssi.GetServices()))
-
-	tmpFile, err := writeSwaggerFile(cfg)
-	if err != nil {
-		logrus.WithError(err).Warnf("unable to write swagger file, skipping handler")
-	} else {
-		httpServer.RegisterPreShutdownHook(func(_ context.Context) error {
-			logrus.Infof("removing temp file %q", tmpFile.Name())
-			err := os.Remove(tmpFile.Name())
-			if err != nil {
-				logrus.WithError(err).Warnf("unable to delete %q during shutdown", tmpFile.Name())
-			}
-			return nil
-		})
-
-		engine.StaticFile("swagger.yaml", tmpFile.Name())
-		engine.GET(SwaggerPrefix, ginswagger.WrapHandler(swaggerfiles.Handler, ginswagger.URL("/swagger.yaml")))
-	}
+	engine.StaticFile("swagger.yaml", "./doc/swagger.yaml")
+	engine.GET(SwaggerPrefix, ginswagger.WrapHandler(swaggerfiles.Handler, ginswagger.URL("/swagger.yaml")))
```

Comment:
Removing this makes it difficult to specify the server urls dynamically. This is important to actually have a nice swagger config.

Reply:
yes, but doing it with a template file is not a great solution because it leaves ugly template files to render on the website

using swaggo's default override system as we had before doesn't have this issue, so I will re-add that

Reply:
As long as there is a way of handling that use case, then this lgtm.